src/components/applications/analytics-visualization.tsx

```
@@ -0,0 +1,531 @@
+            strokeWidth={2}
+            animationDuration={config.animation ? 1500 : 0}
+          >
+            {processedData.map((entry, index) => (
+              <Cell key={`cell-${index}`} fill={entry.fill} />
+            ))}
+          </Treemap>
+        );
+
+      case VisualizationType.GAUGE:
+        const gaugeData = processedData[0];
+        const percentage = gaugeData ? (gaugeData.value / 100) * 180 : 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)